jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -124,7 +118,7 @@ public String getAbsoluteSourceFileString() {
    * @return the extraction path
    */
   public String getAbsoluteExtractionPathString() {
```

Comment:
The Javadoc says "The path is made absolute first", but `extractionPath` is always absolute. Let's update it.